src/js/api/config.js

```diff
@@ -75,7 +75,7 @@ define([
         __webpack_public_path__ = config.base;
         config.width  = _normalizeSize(config.width);
         config.height = _normalizeSize(config.height);
-        config.flashplayer = config.flashplayer || utils.getScriptPath('jwplayer.js') + 'jwplayer.flash.swf';
+        config.flashplayer = config.flashplayer || config.base + 'jwplayer.flash.swf';
```

Comment:
If bundling is the concern this should read:

`config.flashplayer = config.flashplayer || (utils.getScriptPath('jwplayer.js') || config.base) + 'jwplayer.flash.swf'; `